README.md

```diff
@@ -31,7 +31,8 @@ cargo install --git https://github.com/boxdot/gurk-rs gurk
 
 ## Arch Linux
 
-Packaged in the AUR: `gurk-git` and `gurk-bin`
+- Official repository: [`gurk`](https://archlinux.org/packages/extra/x86_64/gurk)
+- AUR: [`gurk-bin`](https://aur.archlinux.org/packages/gurk-bin) and [`gurk-git`](https://aur.archlinux.org/packages/gurk-bin)
```

Comment:
```suggestion
- Official repository, stable version: [`gurk`](https://archlinux.org/packages/extra/x86_64/gurk)
- AUR source build from Git HEAD: [`gurk-git`](https://aur.archlinux.org/packages/gurk-bin)
```

Since the package is prebuilt in official repos the `-bin` package variant should not exist in the AUR and I've filed a deletion request for it. Skip mentioning that in these instructions because it will be obsolete soon.